<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerWindow</class>
 <widget class="QMainWindow" name="CustomerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Banking System - Customer Dashboard</string>
  </property>
  <property name="styleSheet">
   <string>QMainWindow {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                               stop: 0 #f8f9fa, stop: 1 #e9ecef);
}

QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                               stop: 0 #007bff, stop: 1 #0056b3);
    border: none;
    border-radius: 8px;
    color: white;
    font-weight: bold;
    font-size: 14px;
    padding: 10px 20px;
    min-width: 120px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                               stop: 0 #0099ff, stop: 1 #007bff);
}

QListWidget {
    background: white;
    border: 1px solid #dee2e6;
    border-radius: 8px;
    font-size: 14px;
    padding: 5px;
}

QListWidget::item {
    padding: 10px;
    border-bottom: 1px solid #f1f3f4;
}

QListWidget::item:selected {
    background: #e3f2fd;
    color: #1976d2;
}

QLabel {
    color: #495057;
    font-size: 16px;
}

QGroupBox {
    font-weight: bold;
    font-size: 16px;
    color: #343a40;
    border: 2px solid #dee2e6;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px 0 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="leftLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="welcomeLabel">
        <property name="text">
         <string>Welcome, Customer!</string>
        </property>
        <property name="styleSheet">
         <string>QLabel {
    font-size: 20px;
    font-weight: bold;
    color: #007bff;
    margin-bottom: 10px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="actionsGroupBox">
        <property name="title">
         <string>Account Actions</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="viewAccountsButton">
           <property name="text">
            <string>üìã View All Accounts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="transferButton">
           <property name="text">
            <string>üí∏ Card to Card Transfer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="changePasswordButton">
           <property name="text">
            <string>üîë Change Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="generateDynamicPasswordButton">
           <property name="text">
            <string>üé≤ Generate Dynamic Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateInfoButton">
           <property name="text">
            <string>‚úèÔ∏è Update Personal Info</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="logoutButton">
           <property name="text">
            <string>üö™ Logout</string>
           </property>
           <property name="styleSheet">
            <string>QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                               stop: 0 #dc3545, stop: 1 #c82333);
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                               stop: 0 #e45563, stop: 1 #dc3545);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout">
      <item>
       <widget class="QGroupBox" name="accountsGroupBox">
        <property name="title">
         <string>Your Bank Accounts</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="accountsListWidget">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="detailsGroupBox">
        <property name="title">
         <string>Account Details</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="detailsLabel">
           <property name="text">
            <string>Select an account to view details</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string>QLabel {
    background: #f8f9fa;
    border: 1px solid #dee2e6;
    border-radius: 4px;
    padding: 15px;
    font-family: 'Courier New', monospace;
    font-size: 12px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionView_Accounts"/>
    <addaction name="actionTransfer"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionChange_Password"/>
    <addaction name="actionUpdate_Info"/>
   </widget>
   <addaction name="menuAccount"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionView_Accounts">
   <property name="text">
    <string>View Accounts</string>
   </property>
  </action>
  <action name="actionTransfer">
   <property name="text">
    <string>Transfer Money</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionUpdate_Info">
   <property name="text">
    <string>Update Personal Info</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
